fastlane_core/lib/fastlane_core/cert_checker.rb

```diff
@@ -15,11 +15,11 @@ def self.is_installed?(path)
       installed?(path)
     end
 
-    def self.installed_identies
+    def self.installed_identities
```

Comment:
Even though there is a spelling mistake, we can't just rename this class, as it would break the calls from older version of `sigh` when installing the latest `fastlane_core`. Instead we could just have an Ruby `alias_method` for the old method.